### PR TITLE
Components

### PR DESCRIPTION
**Pull request by Mujeeb4582**
## Task done
- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called components.
- Inside components, create a new Calculator.js file.
 ## Live Link 
- [Live Link](https://mujeeb4582.github.io/Math_Megicians/)

### CODE REVIEW

File: src/componenets/calculator.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            <button className="btn" type="button">3</button>
+            <button className="btn color" type="button">&#43;</button>
+          </div>
+          <div className="btn-last-row">
+            <button className="btn zero" type="button">0</button>
+            <button className="btn" type="button">2</button>
+            <button className="btn color" type="button">&#61;</button>
+          </div>
+        </div>
+      </div>
```
Comment (Felix45) [inline src/componenets/calculator.js]: - [ ] Please consider removing the white space on the right and left sides of the buttons on the calculator. It's important to try as much as possible to replicate the design provided for this Milestone, in real-world projects you'll be required to do exactly this. Please see the screen shots below for more details on this.
You can do this by placing the `.screen` div inside the `.btn-container` as illustrated below, also remove the `width` styling for the `.screen` div
```HTML
<div class="calculator-wrapper">
  <div class="btn-container">
    <div class="screen"><p>0</p></div>
  </div>
</div>
```

```CSS
.screen {
    height: 4rem;
    background-color: #444;
    font-size: $myFontSize;
    font-weight: $myFontWeight;
    text-align: right;
    padding: 10px 20px;
    color: aliceblue;
    box-shadow: $shadow;
  }
```
## Microverse design


## Current implementation

File: src/App.js [hunk 2]
```diff
@@ -1,27 +1,12 @@
-import logo from './logo.svg';
-import './App.css';
+import './App.scss';
+import Calculator from './componenets/calculator';
 
 function App() {
   return (
     <div className="App">
```
Comment (Felix45) [inline src/App.js]: - [ ] Please consider changing this functional component to a `class` component, the project requirements for this project require that all components should be class-based components. Please see the screenshot below.

Screenshot from 2022-11-30 19-25-19

You can do this as shown below
```JavaScript
class App extends Component {
  render() {
    return {
      <Calculator />
    };
  }
}
```

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @Mujeeb4582 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :heavy_check_mark: 
- [ ] All linter checks are setup and passing as expected :white_check_mark: 
- [ ] Good descriptive commit messages :white_check_mark: 
- [ ] Git flow is correctly used :white_check_mark: 

almost-there

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Mujeeb, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**